### DISCUSSION TITLE
Remove `ref_*` Attributes

### DISCUSSION THREAD

**Discussion by Aniak5** [ocsf/ocsf-schema#310] [Proposals]
@rroupski and I discussed removing all the original event source attributes (`ref_*` as seen below) and allowing them to be specified in the `unmapped` base event attribute. This would simplify the question of where event source specific data should go as well as reduce the number of source specific attributes. 

![image](https://user-images.githubusercontent.com/6465263/196529951-0523e4df-0f44-4c91-a5e5-c9f55b8e02e4.png)

**Comment by jp-harvey** [comment]
I'm voting to remove them but wanted to add some additional context: `ref_time` is probably important enough to be retained but named to something like `original_time` and be changed to optional.

**Comment by oshezaf** [comment]
I find standardizing the naming for the original fields useful, espeically for a human user.
